_ideas/heatmap/intermediate/1-add-legend.md

```diff
@@ -0,0 +1,14 @@
+## Heatmap - Intermediate
+
+## Task
+Add a legend to the heatmap showing how many posts each shade of blue represents.
+
+## Overview
+To add a legend to the heatmap, you'll have to first look into how the [`Heatmap`](../../../frontend/src/components/Heatmap.js) component is implemented.
+
+The heatmap itself is created using [D3.js](https://d3js.org/), with the bulk of the implementation happening inside the `useEffect` function. You may find looking at [this example](http://bl.ocks.org/tjdecke/5558084) helpful, as it includes a legend.
```

Comment:
I think this is a better reference, because it uses d3 v4 instead of v3. (search for `legend` in your link vs the `legend_g` in my link) http://bl.ocks.org/ganezasan/dfe585847d65d0742ca7d0d1913d50e1

Reply:
I'll swap out the links